spec/models/submission_spec.rb

```diff
@@ -12,4 +14,10 @@
       expect(Submission.unsent).not_to eql nil
     end
   end
-end
\ No newline at end of file
+
+  describe '#enqueue' do
+    it 'sent into delivery' do
+
+    end
```

Comment:
The way I understand this, this is supposed to test the good path of submitting a mailing -> it will be send if validations don't fail. But I agree with you, this could be expressed somehow more obvious. A more explicit description could already be a start 😉